CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+You will find here a summary of the different things you should do / look up to contribute to the project.
+
+## Starting
+
+* First, [fork](https://github.com/ArkScript-lang/Ark/fork) the repository
+* Then, clone your fork: `git clone git@github.com:username/Ark.git`
+* Create a branch for your feature: `git checkout -b feat-my-awesome-idea`
```

Comment:
Recommended that branch types are separated from the branch slug by a forward slash, as follows: 
`feat/my-awesome-idea`. It will help distinguish the "folders" in any git GUI